x-pack/plugins/lens/public/datasources/form_based/dimension_panel/dimension_editor.tsx

```diff
@@ -163,6 +167,32 @@ export function DimensionEditor(props: DimensionEditorProps) {
     [hasRandomSamplingToastFired, layerId, props.notifications.toasts, state.layers]
   );
 
+  const fireOrResetRankingToast = useCallback(
+    (newLayer: FormBasedLayer) => {
+      if (hasRiskyColumnRanking(state.layers[layerId]) && !hasRiskyColumnRanking(newLayer)) {
+        props.notifications.toasts.add({
+          title: i18n.translate('xpack.lens.uiInfo.rankingResetTitle', {
+            defaultMessage: 'Top values ranking reset',
+          }),
+          text: i18n.translate('xpack.lens.uiInfo.rankingResetToAlphabetical', {
+            defaultMessage: 'The use of decimals for this function is not supported for ranking.',
```

Comment:
Maybe something like "Ranking reset to alphabetical rank. Please only use whole numbers to rank by percentile."

Reply:
Updated with latest suggestions by @gchaps